### ISSUE TITLE
Test project does not belong in the source code

### ISSUE THREAD

**Issue by Andries-Smit** [mendix/AppStoreWidgetBoilerplate#51]
All widgets have test projects. That is a really good thing. Much of an improvement.

However a test project... is not code, and to keep the commit clean, the test projects should actually not part of the widget source repository. 

Mendix projects have their own SVN repository where they belong, but they are not publicly accessible for others.

Is there a good solution for splitting the 2?

**Comment by JelteMX** [comment]
Nope. We're going to create a new one for Mendix 6, but this should be a part of it in my opinion. Yes, a lot of widgets are created for a particular project and don't need a test-project, but for those that go in the AppStore we'd like one so we can test it's functionalities. That's why it should be part of the repository.

You have other suggestions?

**Comment by JelteMX** [comment]
And the test-project is not part of the source code, in that regard. Like package.json and Grunt/Gulp are not part of the source code. The source code lives in the `src` folder

**Comment by Andries-Smit** [comment]
They are binary files, so they don't add anything on the diffs. 
The biggest challenge is that they pollute the commits. Lots of .svn stuff .jar files and .java files etc.

What I do like about it that they are matching with the version. 

Ideally they would live in their own SNV repo that is public and connected with the package.
